client/src/components/pages/PageNotFound/style.css

```diff
@@ -0,0 +1,41 @@
+.not-found {
+    min-height: calc(100vh - calc(129px + 70px));
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.not-found--msg {
+    display: block;
+    font-size: 26px;
+    font-family: 'Merriweather', serif;
+    color: #333;
+}
+
+.not-found--code {
+    font-size: 200px;
+    color: var(--primary-color, pink);
+}
+
+.not-found--back-home, .not-found--back-home:hover {
```

Comment:
Better to make a button and change the color to be different from the footer, maybe #505050 :smiley: